src/faq.md

```diff
@@ -202,6 +202,20 @@ Inside or outside of Google, [every Flutter app][FlutterShowcase] uses Dart.
 
 ## Native execution
 
+### Q. Is Dart single-threaded?
+
+No. On native targets, Dart features like isolates and async-await
+can enable multiple threads of execution at any given time.
+The Dart VM will utilize multiple cores of a multi-core processor
+to run those threads concurrently.
+
+[Dart's concurrency architecture](/guides/language/concurrency)
+abstracts the complex, error-prone code of typical shared-memory threading,
+which might explain the misconception that Dart is single-threaded.
```

Comment:
**suggestion:** This reads like it's burying the lede.

```suggestion
The misconception that Dart runs single-threaded might come from its abstraction of shared-memory threading. [Dart's concurrency architecture](/guides/language/concurrency) simplifies threading.

```

Reply:
Gonna keep as is, imo the lede is Dart's great concurrency architecture